eqcorrscan/utils/plotting.py

```diff
@@ -1172,7 +1171,7 @@ def threeD_seismplot(stations, nodes, size=(10.5, 7.5), **kwargs):
     return fig
 
 
-def noise_plot(signal, noise, normalise=False, **kwargs):
+def noise_plot(signal, noise, normalise=False, plotdir=None, **kwargs):
```

Comment:
Please remove `plotdir` argument, it should be handled by the `savefile` kwarg.

Reply:
Ok.